src/markdown/joinUs.md

```diff
@@ -1,11 +1,21 @@
 ---
 heading: Join Us
 meta_page_description: "Monash Human Power application and recruitment page"
-buttons:
-  - buttonText: Apply for Rider Dev
-    link: "https://forms.gle/w1D8tzLVDUgdsJa98"
-  - buttonText: Apply for other teams
-    link: "https://forms.gle/fRsfEiXTxbA2gPfw8"
+eoi_link: "" # TODO: EOI link when it exists
```

Comment:
Could we move this under the general recruitment category (Rider dev application is always open, so EOI is only applicable for general recruitment.